presto-hive-common/src/main/java/com/facebook/presto/hive/zorder/ZOrder.java

```diff
@@ -33,7 +33,7 @@
 {
     public static final int MAX_INPUT_DIMENSIONS = 10;
 
-    // Assume encodingBits is 1-based
+    // Assume encodingBits is 0-based
     private final List<Integer> encodingBits;
```

Comment:
So now, value 5 means we going to use 6 bits to encode the dimension?

Reply:
Yes. The reason why this was changed is because I would still like the maximum value to be `(1 << encodingBits) - 1`, and now the minimum value has changed from `0` to `-(1 << encodingBits)`.